source/content/guides/launch/04-domains.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Launch Essentials
+subtitle: Connect a Domain Name
+description: Learn how to connect your domain to your Pantheon-hosted site.
+launch: true
+anchorid: domains
+generator: pagination
+layout: guide
+categories: [go-live]
+tags: [dns, https, launch, webops]
+type: guide
+pagination:
+    provider: data.launchpages
+use:
+    - launchpages
+    - docs_tags
+permalink: docs/guides/launch/domains/
+nexturl: guides/launch/configure-dns/
+nextpage: Configure DNS and HTTPS
+previousurl: guides/launch/plans/
+previouspage: Upgrade Site Plan
+editpath: launch/03-domains.md
+image: getting-started-Largethumb
+reviewed: "2020-08-03"
+---
+
+This section provides information on how to connect a bare domain and `www` domain to Pantheon's Live environment.
```

Comment:
Throughout the doc, you alternate between "bare domain" and "example.com domain".  I recommend choosing one and using it throughout.

Reply:
I can only find one instance of example.com and is used as an example of what a bare domain looks like.